start.html

```diff
@@ -178,145 +181,341 @@ <h3 class="accordion">
+                         reasonable set of repos
+                         in <span class="c_snip_inline">/etc/yum.repos.d</span>.
+                         For example, for RHEL8, you will need the CRB
+                         (Code Ready Builder) repo; for CentOS 8, you
+                         will need the Centos-PowerTools repo.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         In general, you will need some Perl packages and a lot
+                         of Python3 packages, in addition to some basic packages
+                         that the pbench-agent spec file lists.
+                       </p>
+                     </div>
+                   </div>
+<!-- Section: Installing using ansible  -->
+                   <div class="accordionCard">
+		     <h3 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="false">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text">Installing using ansible</span>
+		       </button>
+		     </h3>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+                       <p class="info_text"  style="text-align:justify">
+                         Once you have a good set of repos installed on your machines, then you can
+                         install <span class="c_snip_inline">pbench-agent</span>. The recommended method for installation is through
+                         ansible playbooks.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         The steps are:
+                       </p>
+                       <div class="info_text" style="text-align:justify">
+                           <li>Make sure that you have ansible and ansible-galaxy installed on your
+                             laptop (or wherever you decide to run the playbooks).</li>
+                           <li>Install the ansible roles from Ansible Galaxy.</li>
+                           <li>Tell ansible where to find these roles.</li>
+                           <li>Create a playbook (you can copy the one below).</li>
+                           <li>Create an inventory file naming the hosts on which you wish to install
+                             <span class="c_snip_inline">pbench-agent</span> and including information about where the config file and ssh key
+                             file can be found.</li>
+                           <li>Run the playbook.</li>
+                       </div>
+                       <p class="info_text"  style="text-align:justify">
+                         The playbook will install the pbench repo and it will then use it to
+                         install the pbench-agent RPM; finally, it will install the
```

Comment:
This reference to `pbench-agent` does not have the `span` wrapping, is that significant?

Reply:
No. Fixed.